### ISSUE TITLE
Update Project Profile: Home Unite Us Add Lola Sarumi

### ISSUE THREAD

**Issue by essencegoff** [hackforla/website#7488]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `home-unite-us.md` file.
- [x] Observe the existing syntax of the front matter block [^1] in the file.
- [x] Find the `leadership` variable and add the following profile. 
```
  - name: Lola Sarumi
    github-handle: lola3736
    role: Lead Product / Project
    links:
      slack: https://hackforla.slack.com/team/U06TAL88YUV
      github: https://github.com/lola3736
    picture: https://avatars.githubusercontent.com/lola3736
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] Home Unite Us page [^2]
- [ ] Once your pull request is merged, go to the initiating ER [^3]
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move initiating ER [^3] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/home-unite-us
[^3]: initiating ER:  #7362

**Comment by HackforLABot** [comment]
Hi @chrjl, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by chrjl** [comment]
i. Availability: asynchronous/evenings
ii. ETA: 1 week
